src/lib/es2020.intl.d.ts

```diff
@@ -56,7 +56,7 @@ declare namespace Intl {
      *
      * [MDN](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Intl#Locale_negotiation).
      */
-    type RelativeTimeFormatLocaleMatcher = "lookup" | "best fit";

```

Comment:
This type must not be renamed, as that would be a breaking change.

Reply:
All existing type modifications were returned, and only cases where the existing type name was not applied were left. If there are any issues with this item, I will reinstate it.